common/structures/instrument.py

```diff
@@ -49,10 +83,10 @@ def gsaoi_mode():
         def nifs_mode():
             return 'ifu'
 
-        instrument_lookup = { 
+        instrument_lookup = {
                             'GMOS-S': gmos_mode,
                             'GMOS-N': gmos_mode,
```

Comment:
Since GMOS-N and GMOS-S have a lot of differences (e.g. different filters), we may have to separate these out at some point, but I don't think we need to worry about that right now.